jobs/haproxy/templates/keepalived.config.erb

```diff
@@ -0,0 +1,35 @@
+global_defs {
+	# Keepalived process identifier
+	lvs_id <%= p('keepalived.lvs_id') %>
+}
+# Script used to check if HAProxy is running
+vrrp_script <%= p('keepalived.healthcheck_name') %> {
+	script "<%= p('keepalived.healthcheck_command') %>"
+	interval 2
+	weight 2
+}
+# Virtual interface
+# The priority specifies the order in which the assigned interface to take over in a failover
+vrrp_instance <%= p('keepalived.vrrp_instance_name') %> {
+	<% if p('keepalived.masterip').include? spec.networks.marshal_dump.first[1].ip %>
```

Comment:
ok then, misinterpretation on my side.

Therefore, we need to take into consideration that we need to configure 1 VIP per AZ. So we should change the parameter keedaplived.vip to a list or a dictionary for deployment to iterate on ?